packages/cli/src/commands/update.ts

```diff
@@ -12,22 +12,19 @@ export async function update(
   const keys =
     keysInput === undefined ? Object.keys(entries) : keysInput.split(",")
 
-  for (let key of keys) {
+  const updateByKey = async (key: string) => {
     if (!(key in entries)) {
       throw new Error(`Key ${key} not set in polkadot-api config`)
     }
 
-    const spinner = ora(`Updating ${key}`).start()
-
     // Exclude metadata file from the entry, otherwise getMetadata would load from the file
     const { metadata: filename, ...entry } = entries[key]
     if (!filename) {
       if (keysInput !== undefined) {
         console.warn(`Key ${key} doesn't have a metadata file to update`)
       }
 
-      spinner.stop()
-      continue
+      return
     }
 
     const metadata = await getMetadata(entry as EntryConfig)
```

Comment:
I think we need to be careful, this function is not prepared to be run in parallel - It's creating [one webworker](https://github.com/polkadot-api/polkadot-api/blob/main/packages/cli/src/metadata.ts#L27) with [one instance of smoldot](https://github.com/polkadot-api/polkadot-api/blob/main/packages/cli/src/smolldot-worker.ts) every time it's being called.

It probably works with a low amount of chains, but it might blow up on resources with a bigger amount.

This part needs to be improved, also the list of "known chains" is [hardcoded ](https://github.com/polkadot-api/polkadot-api/blob/main/packages/cli/src/well-known-chains.ts) to the few we knew back then